Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/core/impl/IndexService.java

```diff
@@ -120,4 +120,20 @@ public IndexingEvent getTotalNumberOfWordsIndexed() {
         return dao.getTotalNumberOfWordsIndexed();
     }
 
+    /**
+     * This method updates the concept in lucene index based on id of the
+     * concept
```

Comment:
important thing here is that the concept is updated in lucene, not that its id is used.